requirements.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with Python 3.10
+# This file is autogenerated by pip-compile with Python 3.8
```

Comment:
🤔

Reply:
Some strange behavior with venv and pyenv... regenerated with 3.10 just in case.